DevilDaggersAssetEditor/Content/dd/Model Bindings.json

```
@@ -1,329 +1,329 @@
 			"?"
 		]
 	},
 	{
 		"AssetName": "boid_pop",
-		"Description": "?",
+		"Description": "Unused?",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "dagger",
-		"Description": "?",
+		"Description": "Pleb lobby dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "dagger1",
-		"Description": "?",
+		"Description": "Level 1 dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "dagger2",
-		"Description": "?",
+		"Description": "Level 2 dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "dagger3",
-		"Description": "?",
+		"Description": "Level 3 regular dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "dagger4",
-		"Description": "?",
+		"Description": "Level 3 homing dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "dagger5",
-		"Description": "?",
+		"Description": "Level 4 regular dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "dagger6",
-		"Description": "?",
+		"Description": "Level 4 homing dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "daggerbronze",
-		"Description": "?",
+		"Description": "Bronze lobby dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "daggerdevil",
-		"Description": "?",
+		"Description": "Devil lobby dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "daggergold",
-		"Description": "?",
+		"Description": "Golden lobby dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "daggerlevi",
-		"Description": "?",
+		"Description": "Leviathan lobby dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "daggersilver",
-		"Description": "?",
+		"Description": "Silver lobby dagger",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "gib",
-		"Description": "?",
+		"Description": "Gems (Including enemies gems)",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "gibegg",
-		"Description": "?",
+		"Description": "Blinking effect of gems close to active spiders",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "gigaegg",
-		"Description": "?",
+		"Description": "Spider 2 egg goo",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "hand",
-		"Description": "?",
+		"Description": "Level 1 Hand",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "hand0",
-		"Description": "?",
+		"Description": "Lobby hand",
 		"Tags": [
-			"?"
+			""
 		]
 	},
 	{
 		"AssetName": "hand2",
-		"Description": "?",
+		"Description": "Level 2 hand",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "hand3",
-		"Description": "?",
+		"Description": "Level 3 hand",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "hand4",
-		"Description": "?",
+		"Description": "Level 3 hand flash",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "hand5",
-		"Description": "?",
+		"Description": "Level 4 hand",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "hand6",
-		"Description": "?",
+		"Description": "Level 4 hand flash",
 		"Tags": [
 			"?"
 		]
 	},
 	{
 		"AssetName": "immortaleye",
-		"Description": "?",
+		"Description": "Orb / Eye",
```

```suggestion
		"Description": "Orb",
```

We don't actually use "Eye" anywhere.